src/components/ItemList/index.jsx

```diff
@@ -26,9 +33,26 @@ function ItemList({ itemName, docId, lastPurchasedDate }) {
   const isDateValid = (date) => {
     if (date) return true;
   };
+  const MarkProductPurchased = (
```

Comment:
remember that only components should begin with a capital letter, is there a reason this should be capitalized?

Reply:
Hello, if I should comply with the camelCase, it was like this from the last issue and I didn't realize it  :)  Thank you very much I already made the change.